src/test/java/gumtree/spoon/AstComparatorTest.java

```diff
@@ -1526,7 +1526,7 @@ public void test_issue59() throws Exception {
 		result.debugInformation();
 
 		assertEquals(1, actions.size());
-		assertTrue(result.containsOperation(OperationKind.Insert, "THROWS", "java.lang.Exception"));
+		assertTrue(result.containsOperation(OperationKind.Insert, "THROWN_TYPES", "ThrownTypes"));
```

Comment:
This has been changed because now we insert a virtual element if no exceptions exist initially. Thus, only `Exception` is not inserted as a child but an entire `CtVirtualElement` is inserted, one of whose children is `Exception.`